commons/pom.xml

```diff
@@ -61,6 +66,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>jakarta.ws.rs</groupId>
+      <artifactId>jakarta.ws.rs-api</artifactId>
+      <version>3.1.0</version>
+    </dependency>
```

Comment:
Is this dependency only needed for `jakarta.ws.rs.core.MediaType`? If so, I think we should rather use string literals (i.e. `application/json`) instead, rather than adding this.